src/features/game/events/sell.ts

```diff
@@ -43,7 +48,7 @@ export function sell({ state, action }: Options): GameState {
       .toDecimalPlaces(18, Decimal.ROUND_DOWN),
     inventory: {
       ...state.inventory,
-      [crop.name]: cropCount.sub(1 * action.amount),
+      [sellable.name]: cropCount.sub(1 * action.amount),
```

Comment:
Is `cropCount` a good name here now that there are cakes that are sellable? Maybe `itemCount`?